akvo/rsr/static/scripts-src/my-results/utils.js

```diff
@@ -286,91 +286,67 @@ export function fullUpdateVisibility(update, activeFilter) {
     // to be interacted with
     // returns true if the update should be fully visible
     let visible;
-    switch (activeFilter) {
-        case c.FILTER_NEED_REPORTING: {
-            visible = [c.UPDATE_STATUS_NEW, c.UPDATE_STATUS_DRAFT, c.UPDATE_STATUS_REVISION];
-            break;
-        }
-        case c.FILTER_SHOW_PENDING: {
-            visible = [c.UPDATE_STATUS_PENDING];
-            break;
-        }
-        case c.FILTER_SHOW_APPROVED: {
-            visible = [c.UPDATE_STATUS_APPROVED];
-            break;
-        }
-        default: {
-            // full visibility to all updates if no filter is active
-            return true;
-        }
+    switch(activeFilter) {
+        case c.FILTER_NEED_REPORTING:
+            {
+                visible = [c.UPDATE_STATUS_NEW, c.UPDATE_STATUS_DRAFT, c.UPDATE_STATUS_REVISION];
+                break;
+            }
+        case c.FILTER_SHOW_PENDING:
+            {
+                visible = [c.UPDATE_STATUS_PENDING];
+                break;
+            }
+        case c.FILTER_SHOW_APPROVED:
+            {
+                visible = [c.UPDATE_STATUS_APPROVED];
+                break;
+            }
+        default:
+            {
+                // full visibility to all updates if no filter is active
+                return true;
+            }
     }
     return visible.indexOf(update.status) > -1;
 }
 
-function tree(model, parentId) {
-    // Construct a tree representation of the subtree of data with object model[parentId] as root
-    //TODO: refactor, we shouldn't need findChildren here
-    const ids = findChildren(parentId, model).ids;
-    const childModel = childModelName(model);
-    const children = ids.map(cId => {
-        return tree(childModel, cId);
-    });
-    if (children.length > 0) {
-        return { id: parentId, model: model, children: children };
-    } else {
-        return { id: parentId };
-    }
-}
-
 export function flatten(arr) {
     // Flatten an array of arrays
     return arr.reduce((acc, val) => acc.concat(Array.isArray(val) ? flatten(val) : val), []);
 }
 
-function keysList(node, open, MEManagerKeys) {
-    // "Disassemble" the tree representation of the data from tree() and return a list of objects.
-    // Each object holds the collapseID of the corresponding Collapse and its activeKey
-    const key = {
-        collapseId: collapseId(node.model, node.id),
-        activeKey: open
-            ? idsToActiveKey(node.children.map(child => child.id.toString()))
-            : MEManagerKeys || []
-    };
-    const children = node.children.filter(child => child.model !== undefined);
-    const childKeys = children.map(node => keysList(node, open));
-    return flatten([key].concat(childKeys));
-}
-
-export function toggleTree(model, id, open, MEManagerKeys) {
-    const fullTree = tree(model, id);
-    return keysList(fullTree, open, MEManagerKeys);
+export function toggleTree(open, MEManagerKeys) {
```

Comment:
The MEManagerKEys parameter is not needed any longer. Also check for calls and make sure they don't include this param.